pkg/apis/radix/v1/radixapptypes.go

```diff
@@ -920,8 +920,8 @@ type RadixPrivateImageHubCredential struct {
 // RadixVolumeMount defines an external storage resource.
 type RadixVolumeMount struct {
 	// Type defines the storage type.
-	// Deprecated, use BlobFuse2 or AzureFile instead.
-	// +kubebuilder:validation:Enum=blob;azure-blob;azure-file;""
+	// Deprecated, use BlobFuse2 instead.
+	// +kubebuilder:validation:Enum=blob;azure-blob;""
 	// +optional
```

Comment:
```suggestion
	// Deprecated: use BlobFuse2 instead.
	// Type defines the storage type.
	// +kubebuilder:validation:Enum=blob;azure-blob;""
	// +optional
```
Deprecation of fields is done when the first line in the comments section starts with "Deprecated: ". Only then will the field appear as deprecated in the code.

Reply:
Fixed